internal/pkg/service/stream/storage/statistics/repository/stats_reset.go

```diff
@@ -0,0 +1,42 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/level"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/statistics"
+)
+
+// ResetStats sums all statistics for data in target level and saves the sum in _reset key.
+// Local and staging levels are unaffected because those data will be moved to target later.
+func (r *Repository) ResetStats(objectKey fmt.Stringer) *op.AtomicOp[op.NoResult] {
+	ops := op.Atomic(r.client, &op.NoResult{})
+
+	// Object prefix contains all statistics related to the object
+	objectPfx := r.schema.InLevel(level.Target).InObject(objectKey)
+
+	var objectSum statistics.Value
+	var resetSum statistics.Value
```

Comment:
This looks like local variable of `Get statistics of the object` function.

Reply:
i would move it here https://github.com/keboola/keboola-as-code/pull/1877/files#diff-63675a976af654c13fa4a9a3b6698123d388f48822571b243bb4dcef3c8a5213R38 inside `Read` function. It is not used in top level.